bootstrap.sh

```diff
@@ -64,3 +64,23 @@ do
     && cp -n Local.templates/$CONFIG_FILE Local/$CONFIG_FILE \
   )
 done
+
+# Build Yubikit
+YUBIKIT_DIR=Carthage/Checkouts/yubikit-ios
+YUBIKIT_OUT=ThirdParty/YubiKit
+
+rm -rf YUBIKIT_DIR
+rm -rf YUBIKIT_OUT
+
+if [ ! -d "$YUBIKIT_DIR" ]; then
+  git clone https://github.com/jumde/yubikit-ios/ $YUBIKIT_DIR
```

Comment:
Should be doing a shallow clone here, since we don't need any git history for a third party lib (`git clone --depth 1 <url> <dir>`)

Reply:
Also, if you are going to checkout immediately after to a different branch, you should clone it just to that branch:

`git clone -b simulator_release --depth 1 https://github.com/jumde/yubikit-ios/ $YUBIKIT_DIR`

Reply:
resolved!